charts/dbildungs-iam-keycloak/dev-realm-spsh.json

```diff
@@ -417,7 +417,7 @@
     "requiredCredentials": [
       "password"
     ],
-    "passwordPolicy" : "maxAuthAge(0)",
+    "passwordPolicy": "maxAuthAge(0) and passwordHistory(3) and length(8) and digits(1) and upperCase(1) and specialChars(1)",
```

Comment:
Should also contain `lowerCase(1)` (Ticket was missing this, confirmed by Christian)